tests/model/issue.model.spec.ts

```diff
@@ -43,3 +48,112 @@ describe('Issue model class', () => {
         });
     });
 });
+
+describe('Issue', () => {
+    let dummyTeam: Team;
+    let dummyIssue: Issue;
+    let otherDummyIssue: Issue;
+    let dummyIssueWithTeam: Issue;
+
+    let githubComment: GithubComment;
+    let newIssueDispute: IssueDispute;
+    let newTesterResponse: TesterResponse;
+
+    const noReportedDescriptionString = 'No details provided by bug reporter.\n';
+    const tutorResponseStringHeader = '# Tutor Moderation\n\n';
+
+    beforeEach(() => {
+        dummyTeam = new Team({
+            id: 'F09-2',
+            teamMembers: []
+        });
+        dummyIssue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_EMPTY_DESCRIPTION);
+        otherDummyIssue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_ASSIGNEES);
+        dummyIssueWithTeam = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+
+        githubComment = new GithubComment();
+        githubComment.body = 'Sample Text';
+
+        newIssueDispute = new IssueDispute('Cannot Work', 'Help Please');
+        newTesterResponse = new TesterResponse('Cannot Work', 'Help Please', '- [ ] Not Done', 'Reason');
+    });
+
+    it('should be initialized with the correct phase and team with clone()', () => {
```

Comment:
I think we should rephrase the spec description, so it is clear which method is being tested in the spec.
Something like: `it('.clone() should intialise the cloned issue with the correct phase and team')`

This makes it clearer that we are a testing a specific method in the Issue class.

If the test fails, we'll also get a clear idea about the specific method that may be broken:
![image](https://user-images.githubusercontent.com/35621759/99162210-1c36c200-2736-11eb-8044-490e8e6cb30c.png)